tests/test_lidarr.py

```diff
@@ -0,0 +1,967 @@
+        match_querystring=True,
+    )
+    artist = lidarr_client.get_artist(1)
+
+    responses.add(
+        responses.PUT,
+        "https://127.0.0.1:8686/api/v1/artist",
+        headers={"Content-Type": "application/json"},
+        body=load_fixture("lidarr/artist.json"),
+        status=202,
+        match_querystring=True,
+    )
+    data = lidarr_client.upd_artist(data=artist)
+    assert isinstance(data, dict)
+
+
+@pytest.mark.usefixtures
+def test_delete_artist(responses, lidarr_client):
+    responses.add(
+        responses.DELETE,
+        "https://127.0.0.1:8686/api/v1/artist/1",
+        headers={"Content-Type": "application/json"},
+        body=load_fixture("common/delete.json"),
+        status=200,
+        match_querystring=True,
+    )
+    data = lidarr_client.delete_artist(1)
+    assert isinstance(data, dict)
+
+
+@pytest.mark.usefixtures
+def test_get_album(responses, lidarr_client):
+    responses.add(
+        responses.GET,
+        "https://127.0.0.1:8686/api/v1/album?includeAllArtistAlbums=False",
+        headers={"Content-Type": "application/json"},
+        body=load_fixture("lidarr/album_all.json"),
+        status=200,
+        match_querystring=True,
+    )
+    data = lidarr_client.get_album()
+    assert isinstance(data, list)
+
+    responses.add(
+        responses.GET,
+        "https://127.0.0.1:8686/api/v1/album/1?includeAllArtistAlbums=False",
+        headers={"Content-Type": "application/json"},
+        body=load_fixture("lidarr/album.json"),
+        status=200,
+        match_querystring=True,
+    )
+    data = lidarr_client.get_album(albumIds=1)
+    assert isinstance(data, dict)
+
+    responses.add(
+        responses.GET,
+        "https://127.0.0.1:8686/api/v1/album?includeAllArtistAlbums=False&foreignAlbumId=123456-123456",
+        headers={"Content-Type": "application/json"},
+        body=load_fixture("lidarr/album.json"),
+        status=200,
+        match_querystring=True,
+    )
+    data = lidarr_client.get_album(foreignAlbumId="123456-123456")
+    assert isinstance(data, dict)
+
+    responses.add(
+        responses.GET,
+        "https://127.0.0.1:8686/api/v1/album?includeAllArtistAlbums=False&albumids=1&albumids=2&albumids=3",
+        headers={"Content-Type": "application/json"},
+        body=load_fixture("lidarr/album_all.json"),
+        status=200,
+        match_querystring=True,
+    )
+    data = lidarr_client.get_album(albumIds=[1, 2, 3])
+    assert isinstance(data, list)
+
+    responses.add(
+        responses.GET,
+        "https://127.0.0.1:8686/api/v1/album?includeAllArtistAlbums=True&artistId=1",
+        headers={"Content-Type": "application/json"},
+        body=load_fixture("lidarr/album_all.json"),
+        status=200,
+        match_querystring=True,
+    )
+    data = lidarr_client.get_album(artistId=1, allArtistAlbums=True)
+    assert isinstance(data, list)
+
+
+@pytest.mark.usefixtures
+def test__album_json(responses, lidarr_client):
```

Comment:
one too many underscores

Reply:
This one was following the method name e.g. `test_` plus `_album_json` = `test__album_json` or should it not be referenced like this?

Reply:
Not necessarily, unless you have the need to make a functional distinction between two variables named `album_json` and `_album_json`. But in that case, a different naming schema would be more appropriate.

In general it's best to avoid double underscores for anything except [dunder methods](https://www.geeksforgeeks.org/dunder-magic-methods-python/), unless absolutely necessary.